src/mame/rm/rm380z_m.cpp

```diff
@@ -7,7 +7,8 @@ RM 380Z machine
 
 */
 
-
+//#include <iostream>
+//#include <bitset>
```

Comment:
Are these leftovers from something?  At any rate, `#include`ing things before the prefix header is problematic.

Reply:
Yes, leftovers from my debug tracing.  Now removed.  std::bitset is quite useful for printing binary numbers in trace.